### PR TITLE
8255241: [TestBug] Re-enable few ignored tests in javafx.controls module that pass with latest code

### PR DESCRIPTION
**Pull request by aghaisas**
Some of the ignored unit tests in javafx.controls module pass with latest code.
This bug is to address -
- removal of ignored tag
- fixing the test if fix is minor

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8255241](https://bugs.openjdk.java.net/browse/JDK-8255241): [TestBug] Re-enable few ignored tests in javafx.controls module that pass with latest code

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jfx pull/331/head:pull/331`
`$ git checkout pull/331`

### CODE REVIEW

File: modules/javafx.controls/src/shims/java/javafx/scene/control/TableColumnShim.java [hunk 1]
```diff
@@ -30,5 +30,9 @@ public static void setTableView(TableColumn tc, TableView tv) {
         tc.setTableView(tv);
     }
 
+    // NOTE: These constants are direct copy of TableColumnBase
+    public static final double DEFAULT_WIDTH = 80.0F;
```
Comment (kevinrushforth) [inline modules/javafx.controls/src/shims/java/javafx/scene/control/TableColumnShim.java:34]: Can you define these to reference those constants rather than copying them, like this?

```
     public static final double DEFAULT_WIDTH = TableColumnBase.DEFAULT_WIDTH ;
```

If so, you can remove the comments about needing to keep them in sync.

File: modules/javafx.controls/src/main/java/javafx/scene/control/TableColumnBase.java [hunk 2]
```diff
@@ -117,7 +117,8 @@ public void setWidth(TableColumnBase tableColumnBase, double width) {
      *                                                                         *
      **************************************************************************/
 
-    // NOTE: If these numbers change, update the copy of this value in TableColumnHeader
+    // NOTE: If these numbers change, update the copy of this value in
+    // TableColumnHeader and in TableColumnShim
```
Comment (kevinrushforth) [inline modules/javafx.controls/src/main/java/javafx/scene/control/TableColumnBase.java:121]: I think you can revert this change.

**Review by kevinrushforth — COMMENTED** [review]
It works for me on Windows. Have you run it on more than one platform?

I left one suggestion for you regarding the constants in TableColumnShim.

**Comment by aghaisas** [comment]
> It works for me on Windows. Have you run it on more than one platform?

I have run it on macOS and Windows.

> I left one suggestion for you regarding the constants in TableColumnShim.

Thanks for this. That's a cleaner way instead of copying constants. I will make this change soon.

**Review by kevinrushforth — APPROVED** [review]
